pkg/controller/type.go

```diff
@@ -0,0 +1,20 @@
+package controller
+
+import (
+	"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+)
+
+const (
+	BackupConfig     = "backup.appscode.com/config"
+	ContainerName    = "restikbackup"
+	VolumeName       = "restik-volume"
```

Comment:
This will be provided by the user in the TPR if they want to backup inside a disk. This will not be needed when backed up in cloud bucket.